theseus/tests/test_theseus_layer.py

```diff
@@ -139,6 +139,7 @@ def error_fn(optim_vars, aux_vars):
     assert not objective.vectorized
     theseus_layer = th.TheseusLayer(optimizer, vectorize=True)
     assert objective.vectorized
+    objective.disable_vectorization()
```

Comment:
So we are turning vectorization off for the end-to-end unit tests?

Reply:
Yeah, they were too slow because of the extra overhead, since this tests only has one cost function. We could add an extra one where vectorization is turned on.